tests/test-create-delete-idempotent-%image.cmd

```diff
@@ -0,0 +1,5 @@
+footloose config create --config %testName.footloose --name %testName --key %testName-key --image quay.io/footloose/%image
+footloose create --config %testName.footloose
+%out footloose create --config %testName.footloose echo success
```

Comment:
`create` doesn't take any argument like `ssh` does, so `echo success` will not work here.

I think the easier way to do it is to do:

```
footloose create --config %testName.footloose
footloose create --config %testName.footloose
footloose delete --config %testName.footloose
footloose delete --config %testName.footloose
```

If one of those commands fails, the test will fail. If they all succeed, the test will succeed, which seems to the right thing when testing for idempotency.